cmake/FindPYQTSIP.cmake

```diff
@@ -0,0 +1,75 @@
+##########################################################################
```

Comment:
This file should live in CCPAPI/cmake/FindPYQTSIP.cmake. No need to put it at the top level as it is specific to this project.

Reply:
Done!

Reply:
As I mentioned in a later comment about general organization, as a module it should live in **CCPAPI/cmake**.